frameworks/cassandra/universe/config.json

```diff
@@ -730,6 +735,116 @@
+        },
+        "counter_cache_save_period": {
+          "type": "integer",
+          "description": "the amount of time after which Cassandra saves the counter cache (keys only).",
+          "default": 7200
+        },
+        "trickle_fsync_interval_in_kb": {
+          "type": "integer",
+          "description": "The size of the fsync in kilobytes.",
+          "default": 10240
+        },
+        "trickle_fsync": {
+          "type": "boolean",
+          "description": "When set to true, causes fsync to force the operating system to flush the dirty buffers at the set interval ",
+          "default": false
+        },
+        "incremental_backups": {
+          "type": "boolean",
+          "description": "Backs up data updated since the last snapshot was taken. When enabled, Cassandra creates a hard link to each SSTable flushed or streamed locally in a backups subdirectory of the keyspace data.",
+          "default": false
+        },
+        "snapshot_before_compaction": {
+          "type": "boolean",
+          "description": "Enables or disables taking a snapshot before each compaction. A snapshot is useful to back up data when there is a data format change.",
+          "default": false
+        },
+        "cross_node_timeout": {
+          "type": "boolean",
+          "description": "operation timeout information exchange between nodes (to accurately measure request timeouts).",
+          "default": false
+        },
+        "role_manager": {
+          "type": "string",
+          "description": "Part of the Authentication & Authorization backend that implements IRoleManager to maintain grants and memberships between roles.",
+          "default": "CassandraRoleManager"
+        },
+        "commit_failure_policy": {
+          "type": "string",
+          "description": "Policy for commit disk failures.",
+          "default": "stop"
+        },
+        "key_cache_size_in_mb": {
+          "type": "string",
+          "description": "A global cache setting for the maximum size of the key cache in memory (for all tables). ",
+          "default": ""
+        },
+        "counter_cache_size_in_mb": {
+          "type": "string",
+          "description": "When no value is set, Cassandra uses the smaller of minimum of 2.5% of Heap or 50MB.",
+          "default": ""
+        },
+        "commitlog_sync": {
+          "type": "string",
+          "description": "The method that Cassandra uses to acknowledge writes in milliseconds",
+          "default": "periodic"
+        },
+        "index_summary_capacity_in_mb": {
+          "type": "string",
+          "description": "Fixed memory pool size in MB for SSTable index summaries.",
+          "default": ""
+        },
+        "rpc_server_type": {
+          "type": "string",
+          "description": "Cassandra provides three options for the RPC server. sync and hsha performance is about the same, but hsha uses less memory.",
+          "default": "sync"
```

Comment:
please use enums since there are only 3 options

Reply:
Cassandra provides two out-of-the-box options for the RPC Server : sync, hsha.
Alternatively,  can provide your own RPC server by providing the fully-qualified class name
of an o.a.c.t.TServerFactory that can create an instance of it.
@samvantran should I use enum with 2 values sync, hsha or should I keep it as it is.

Reply:
If a user can provide their own server then you can leave as is. This goes for any config that allows for custom values. It's only if there is a finite set do we use enums.